lib/src/MultiPart.cc

```diff
@@ -34,7 +34,15 @@ const std::vector<HttpFile> &MultiPartParser::getFiles() const
 {
     return files_;
 }
-
+std::map<std::string, HttpFile> MultiPartParser::getFilesMap() const
+{
+    std::map<std::string, HttpFile> result;
+    for (auto &file : files_)
+    {
+        result.insert({file.getItemName(), file});
```

Comment:
result.emplace(file.getItemName(),file);
note: the HttpFile instance is copied but it should not be a problem

Reply:
Yes, impl idiom